ci/ansible/roles/pulp-backup/handlers/main.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Restart services
```

Comment:
This doesn't restart any services. It ensures services are started. The former always produces a change of state, and the latter only produces a change of state if needed. How about calling this "Start Pulp"?